integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ConversationStrategy.java

```diff
@@ -0,0 +1,5 @@
+package no.difi.meldingsutveksling.receipt;
+
+public interface ConversationStrategy {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotate the "ConversationStrategy" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1609)